src/services/WorkspaceWorkerPoolApi.ts

```diff
@@ -0,0 +1,52 @@
+import { WorkerPool, WorkerPoolCreate, WorkerPoolEdit, WorkerPoolFilter, WorkerPoolResponse, WorkerFlowRunResponse, WorkerFlowRun, WorkerFlowRuns } from '@/models'
+import { mapper, WorkspaceApi } from '@/services'
+
+export interface IWorkspaceWorkerPoolApi {
+  createWorkerPool: (request: WorkerPoolCreate) => Promise<WorkerPool>,
+  getWorkerPoolByName: (workerPoolName: string) => Promise<WorkerPool>,
+  getWorkerPools: (filter: WorkerPoolFilter) => Promise<WorkerPool[]>,
+  updateWorkerPool: (workerPoolName: string, request: WorkerPoolEdit) => Promise<void>,
+  deleteWorkerPool: (workerPoolName: string) => Promise<void>,
+  getWorkerPoolRuns: (workerPoolName: string, request: WorkerFlowRuns) => Promise<WorkerFlowRun[]>,
+}
+
+export class WorkspaceWorkerPoolApi extends WorkspaceApi implements IWorkspaceWorkerPoolApi {
+
+  protected override routePrefix = '/experimental/worker_pools/'
```

Comment:
Is the `experimental` prefix something specific to worker pool or something we might be seeing in other features?

Reply:
It's a worker pool thing for now
<img width="934" alt="Screenshot 2022-12-19 at 9 54 37 AM" src="https://user-images.githubusercontent.com/40467112/208453829-4dba1814-de7b-461d-8f76-971019ad6535.png">